benches/src/lib.rs

```diff
@@ -546,7 +546,8 @@ impl TryFrom<VmBench> for VmBenchPrepared {
         }
         let storage_diff = vm.storage_diff();
         let mut vm = vm.remove_recording();
-        let mut diff = start_vm.diff(&vm);
+        // TODO: Check if this is the intended use of rollback_to.
```

Comment:
@xgreenx please check that this is how we intend to use the new `rollback_to` funciton, in which case we can remove the TODO

Reply:
I'm pretty sure that's correct